ncm-grub/src/main/perl/grub.pm

```diff
@@ -16,6 +16,7 @@ package NCM::Component::grub;
 #
 
 use strict;
```

Comment:
can you add `use warnings;` or better yet replace everything from here to the top with
`#${PMpre} NCM::Component::grub${PMpost}` and let the templating do its magic